test/lib/lang_test.rb

```diff
@@ -7,7 +7,7 @@ def test_langs_exist
     end
 
     def test_langs_length
-      assert_equal(75, Wovnrb::Lang::LANG.length)
+      assert_equal(77, Wovnrb::Lang::LANG.length)
```

Comment:
I would be on board with deleting all such tests and assertions on this variable 😆